internal/db/schema/migrations/oss/postgres/88/01_remove_session_connection_state.up.sql

```diff
@@ -0,0 +1,221 @@
+-- Copyright (c) HashiCorp, Inc.
```

Comment:
Just a heads up that the migration number will need to be bumped

Reply:
Thanks!